recipes/cylc-flow/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4"
+
+build:
+  number: 0
+  skip: True  # [not linux]
```

For packages which aren't `noarch` this is the way to do it:
```suggestion
  skip: True  # [py<37 or not linux]
```

---

Covered in comment above.